### DISCUSSION TITLE
SDP disable memory attention; what does it do? Should I activate it?

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [vladmandic/sdnext#1924] [Q&A] [answered]
![Capture](https://github.com/vladmandic/automatic/assets/140368252/1908e30c-f97d-4872-a47f-2f92c3bbe759)

does this improve the performance if yes then at what cost?

**Comment by vladmandic** [comment, marked as answer]
it actually slows the performance just a little bit, but makes images reproducible as it removes some randomness from the process.

**Reply by weirdlighthouse** [reply]
👍
